README.md

```diff
@@ -51,38 +51,32 @@ $ yarn start
 ```javascript
 // Import Jumpsuit
 import React from 'react'
-import { Render, State, Actions, Component } from 'jumpsuit'
+import { Render, State, Actions, Component, Effect } from 'jumpsuit'
 
 // Create a state with some actions
 const CounterState = State({
   // Initial State
   initial: { count: 0 },
   // Actions
-  increment ({ count }) {
-    return { count: count + 1 }
-  },
-  decrement ({ count }) {
-    return { count: count - 1 }
-  },
+  increment: ({ count }) => ({ count: count + 1 }),
+  decrement: ({ count }) => ({ count: count - 1 })
 })
 
 // Create an async action
-Effect('incrementAsync' () => {
-  setTimeout(() => {
-    Actions.increment()
-  }), 1000)
-}
+Effect('incrementAsync', () => {
+  setTimeout(() => Actions.increment(), 1000)
+})
 
 // Create a subscribed component
 const Counter = Component({
   render() {
     return (
       <div>
         <h1>{ this.props.count }</h1>
-        <button onClick={ () => Actions.increment() }>Increment</button>
-        <button onClick={ () => Actions.decrement() }>Decrement</button>
+        <button onClick={ Actions.increment }>Increment</button>
+        <button onClick={ Actions.decrement }>Decrement</button>
```

Comment:
I think it's just a lack of discipline. It's functionally the same thing.

People may leave in the parens by accident, but that is not the fault of this pattern.

Whenever you see something like this:

```jsx
<button onClick={() => someFn()}>Click Me</button>
```

^ this can ALWAYS be simplified to:

```jsx
<button onClick={someFn}>Click Me</button>
```